src/Compilers/Core/Desktop/CodeAnalysisDesktopResources.resx

```diff
@@ -132,4 +132,10 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="RuleSetBadAttributeValue" xml:space="preserve">
+    <value>The attribute {0} has an invalid value of {1}.</value>
+  </data>
+  <data name="RuleSetMissingAttribute" xml:space="preserve">
+    <value>The element {0} is missing an attribute named {1}.</value>
+  </data>
```

Comment:
@jaredpar Did you consider the `IXmlLineInfo` interface?
https://msdn.microsoft.com/en-us/library/system.xml.ixmllineinfo(v=vs.110).aspx